build.sbt

```diff
@@ -36,7 +36,9 @@ resolvers ++= List(
 libraryDependencies ++= {
   Seq(
     "org.broadinstitute" %% "lenthall" % lenthallV,
+    "com.google.guava" % "guava" % "20.0",
```

Comment:
I was trying to pull in the library that Cromwell uses to escape strings when creating`URI`s from them just so they had the same behavior.